src/EventTimer.ts

```diff
@@ -32,7 +32,15 @@ interface PageEventStatus {
 }
 
 export class EventTimer {
-	events: Event[];
+	private _events: Event[];
+	private static _externallyDefinedEventNames = [
+		'cmp-tcfv2-init',
```

Comment:
I think it would be worth having a reference to where these come from?

- `got-consent` https://git.io/Jnnyv
- `init` https://git.io/Jnny2
- etc.?

And also add `cmp-***-ui-displayed`, as https://git.io/JnnyO ?

Reply:
Good shout, thanks! Will refactor in the next couple of days. Merging this now 🚀